packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -60,6 +60,9 @@
 import java.util.HashSet;
 import java.util.Set;
 
+import java.util.HashSet;
+import java.util.Set;
+
```

Comment:
Can remove this

Reply:
https://github.com/facebook/react-native/pull/47634/commits/c14e0268fcc10a78bb60fa65a274f34b14e808c2